### DISCUSSION TITLE
How to Increase font size

### DISCUSSION THREAD

**Discussion by emilyanncr** [isontheline/pro.webssh.net#921] [Q&A]
I can't find a setting where I can increase the font size in the terminal.  I've checked under 'Settings' and 'Preferences'.  I searched 'font size' in the help search box but the results do not say how to increase it.  Can you please advise?  Thanks.

**Comment by isontheline** [comment]
Hello @emilyanncr 👋

Thank you a lot for reaching me out 🙏

Font size could be increase by two ways :
* Pinch in / out the terminal
* OR edit your connection > Terminal Settings > Font Size

Could you give them a try?

Have a nice day ☀️

**Comment by emilyanncr** [comment]
Sure thing, thanks!
